app/renderer/js/pages/preference/find-accounts.js

```diff
@@ -0,0 +1,79 @@
+'use-strict';
+
+const { remote, ipcRenderer } = require('electron');
+
+const { dialog } = remote;
+
+const BaseComponent = require(__dirname + '/../../components/base.js');
+
+class FindAccounts extends BaseComponent {
+	constructor(props) {
+		super();
+		this.props = props;
+	}
+
+	template() {
+		return `
+			<div class="settings-card certificate-card">
+				<div class="certificate-input">
+					<div>Organization URL</div>
+					<input class="setting-input-value" value="zulipchat.com"/>
```

Comment:
Do we want this to be a value or placeholder? 
cc @rishig

Reply:
From Priyank's message [here](https://github.com/zulip/zulip-electron/pull/638#discussion_r251202738), we wanted this to be pre-filled with .zulipchat.com as a value earlier, but we've since then decided to change this to zulipchat.com [here](https://github.com/zulip/zulip-electron/pull/638#discussion_r272813020). Either way, we want this to be a pre-filled value.